reana_workflow_controller/rest/workflows.py

```diff
@@ -12,16 +12,15 @@
 from uuid import uuid4
 
 from flask import Blueprint, jsonify, request
+from reana_commons.config import WORKFLOW_TIME_FORMAT
```

Comment:
Given that `WORKFLOW_TIME_FORMAT` was moved to reana-commons, we should remove it from `reana_workflow_controller.config` and update the remaining imports in reana-workflow-controller. What do you think?